pyproject.toml

```diff
@@ -52,6 +51,8 @@ types-paramiko = "^0.1.7"
 types-requests = "^2.25.0"
 types-python-dateutil = "^0.1.4"
 types-PyYAML = "^5.4.3"
+rstcheck = "^3.3.1"
+snooty-lextudio = "^1.9.2-alpha.0"
```

Comment:
Why this? Is is something akin to https://github.com/vscode-restructuredtext/vscode-restructuredtext ?

Reply:
The snooty is a dependency of the restructuredtext extension. If it's not installed, the vscode complains about it, when trying to preview the rst file.